src/gui/plugins/component_inspector/ComponentInspector.cc

```diff
@@ -1016,13 +1032,22 @@ void ComponentInspector::OnLight(
   };
 
   ignition::msgs::Light req;
-
-  // todo(anyone) Use the field isLightOn in light.proto from
-  // Garden on.
-  auto header = req.mutable_header()->add_data();
-  header->set_key("isLightOn");
-  std::string *value = header->add_value();
-  *value = std::to_string(_isLightOn);
+  {
+    // todo(anyone) Use the field isLightOn in light.proto from
+    // Garden on.
+    auto header = req.mutable_header()->add_data();
+    header->set_key("isLightOn");
+    std::string *value = header->add_value();
+    *value = std::to_string(_isLightOn);
+  }
+  {
+    // todo(anyone) Use the field visualize_visual in light.proto from
```

Comment:
this is to be added right? I would suggest just `visualize` to be consistent with the sdf tag.

Reply:
The tricky thing is that protobuf defaults to false when a boolean is unset, so I suspect that adding a `bool visualize` field would cause lights to be invisible by mistake. One workaround could be to use a nested message `Boolean visualize`, so one can check

```
if (!msg.has_visualize())
  visualize = true;
```